act-rules-format.bs

```diff
@@ -52,66 +48,91 @@ The separation between atomic rules and composite rules creates a division of re
 
+<div class=example>
+  <p>ACT Rules in the Auto-WCAG community groups use file names as identifiers. They include a number for a WCAG 2.1 success criterion, and a handle that includes an element name or attribute:</p>
+  <ul>
+    <li>SC1-2-3-video-alternative</li>
+    <li>SC2-2-1+SC3-2-5-meta-refresh</li>
+    <li>SC4-1-1-unique-id</li>
+  </ul>
+</div>
 
-Rule Description {#structure-description}
-=========================================
 
-An ACT Rule MUST have a description that is in plain language and provides a brief explanation of what the rule does.
+Rule Description {#rule-description}
+=========================================
 
+An ACT Rule MUST have a description that is in plain language, and provides a brief explanation of what the rule does.
 
-Accessibility Requirements {#structure-accessibility-requirements}
-==================================================================
+<div class=example>
+  This rule checks that the HTML page has a title
+</div>
 
-<div class=note>
-  **Editor's note:** The ACT Task Force is looking for feedback about the use of the term "pass" in relation to rules. While rules can "pass", their corresponding accessibility requirement can fail. A section has been added to make this explicit in the rule, but we would like to know if this is sufficient.
+<div class=example>
+  The rule checks that for the `html` element, there is no mismatch between the primary language in non-empty `lang` and `xml:lang` attributes, if both are used.
 </div>
 
-Accessibility requirements are just that: A requirement that a particular web page must conform to for it to be considered accessible. This can (and usually does) include the WCAG success criteria. Often organizations have additional requirements which may come from different sources, such as local laws, internal standards, etc. These too are considered accessibility requirements and can be tested using the ACT Rules Format. What the precise requirements are for any test is beyond the scope of the ACT Rules Format.
+Accessibility Requirements {#accessibility-requirements}
+========================================================
 
-[=Atomic rules=] and [=composite rules=] SHOULD identify the accessibility requirements that fail when the outcome of a rule is Fail. An ACT Rule is a complete or partial test for one or more accessibility requirements. This means that most ACT rules test only part of an accessibility requirement, but it MUST NOT test more than the accessibility requirement it lists.
+Accessibility requirements are just that: A requirement that a particular web page must conform to for it to be considered accessible. A common example of accessibility requirements are the WCAG 2.1 success criteria. Often organizations have additional requirements which may come from different sources, such as laws, internal standards, etc. These too are considered accessibility requirements which can be tested using the ACT Rules Format.
```

Comment:
"web page" -> "test subject"
"must conform to" -> "conforms to" (drop the "must")
"Often organizations" -> "Some organizations"
"which may come from" -> "that come from" (drop the "may")

Reply:
Test subjects aren't required to conform to WCAG, web pages are. The way I see it, web pages are a type of test subject. JSX or Vue files are examples of test subjects that are not web pages. Instead, they are used to build a web page. WCAG does not apply to those files, nevertheless those can be tested by ACT Rules. I think this sentence is correct.

Reply:
I don't understand why you want to limit "accessibility requirements" to web page and WCAG only. As I understand it, accessibility requirements is more generic. It can be used for any requirement, for any test subject. We happen to only *currently* use it for WCAG and web pages in Auto-WCAG. Anyway, this sentence does not set a normative requirement and could be edited later, if we decide to.